src/components/modules/banners/contributionsTemplate/ContributionsTemplate.tsx

```diff
@@ -162,9 +163,10 @@ const ContributionsTemplate: React.FC<ContributionsTemplateProps> = ({
     supportingText,
     ticker,
     cta,
+    backgroundColour,
```

Comment:
👍 perhaps one day we can make the styling fully configurable, and integrated with the banner tool

Reply:
yeah for sure - I think the frequency of small tweaks we've had to make is a big argument for getting this in the tool!